app/furniture/tobias/routes.rb

```diff
@@ -0,0 +1,11 @@
+class Tobias
+  class Routes
+    def self.append_routes(router)
```

Comment:
When we call `router.resources`, we are connecting the URL `path` to the application.

`/spaces/:space_id/rooms/:room_id`, nested under the "Section" the TOBIAS is in.

So, `resources :tobiases` creates an entry point into our application at the `/spaces/:space_id/rooms/:room_id/tobiases` path. When we nest `resources` calls within a `do...end` block, like we do with `router.resources :trusts` this nests them within that path , a-la `/spaces/:space_id/rooms/:room_id/tobiases/:tobias_id/trusts` 

What path do you think the line with `router.resources :payouts` creates?